### PR TITLE
move user input to function to remove need for mutable variables

### PR DESCRIPTION
**Pull request by sabjorn**
used the tool to create this PR :D

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -21,6 +22,14 @@ impl fmt::Display for Config {
     }
 }
 
+fn get_user_input() -> String {
```
Comment (antoineMoPa) [inline src/main.rs:25]: @sabjorn maybe we should just use [rprompt](https://docs.rs/rprompt/latest/rprompt/#)?

Comment (sabjorn) [reply src/main.rs:25]: up to you -- I am usually on the side of not pulling in a dependency unless I really need it -- or it provides a better implementation than I can do. BUT it is your project so feel free :D

Comment (antoineMoPa) [reply src/main.rs:25]: I think for now that is good!

Comment (antoineMoPa) [reply src/main.rs:25]: Maybe I'll refactor later.

**Review by antoineMoPa — APPROVED** [review]
LGTM!

**Pull request merged**